tests/kuttl/common/assert_sample_deployment.yaml

```diff
@@ -212,17 +186,6 @@ status:
   availableReplicas: 1
   replicas: 1
 ---
-# the openshift annotations can't be checked through the deployment above
-apiVersion: v1
-kind: Pod
-metadata:
-  annotations:
-    openshift.io/scc: anyuid
-  labels:
-    service: placement
```

Comment:
Why we dropped the Pod assert? I guess we still have the same placement api pod running